### DISCUSSION TITLE
Why isn't Dialogic on the Asset Library?

### DISCUSSION THREAD

**Discussion by VojtaStruhar** [dialogic-godot/dialogic#2397] [General]
I feel like Dialogic is one of the biggest and most well known godot plugins and I was very surprised that it's not available through the Asset Library right from the editor. Why? It would be so convenient.

**Comment by CakeVR** [comment]
The first version (Dialogic 1) is on the Asset Library.
While Dialogic 2 will be on the Asset Library once it properly released.

Be aware: These two iterations of Dialogic are inherently different.

**Reply by VojtaStruhar** [reply]
Is it? I just wrote "Dialogic" into the search and nothing showed up. I'm on 4.3, so maybe it's just not updated or something

**Reply by Jowan-Spooner** [reply]
If you are browsing the asset library inside godot, then plugins for older versions (as dialogic 1.x is only for godot 3.x) won't show up.

**Reply by VojtaStruhar** [reply]
I see, thanks
